src/pages/bootstrap/index.md

```diff
@@ -61,9 +61,9 @@ themes that build on the Bootstrap framework to provide a more customized and st
 
 #### Bootstrap Resources: 
 
-<a href='http://blog.getbootstrap.com/' target='_blank' rel='nofollow'>Bootstrap Blog</a>: Bootstrap's offical blog
-<a href='http://expo.getbootstrap.com/' target='_blank' rel='nofollow'>Bootstrap Expo</a>: Bootstrap site inspiration
-<a href='http://builtwithbootstrap.com/' target='_blank' rel='nofollow'>Built with Bootstrap</a>: Showcase of sites built using Bootstrap
-<a href='https://github.com/twbs/bootlint' target='_blank' rel='nofollow'>Bootlint</a>: HTML linter for projects using Bootstrap
-<a href='https://bootsnipp.com/' target='_blank' rel='nofollow'>Bootsnipp</a>: Design elements and code snippets for Bootstrap
-<a href='http://expo.getbootstrap.com/resources/' target='_blank' rel='nofollow'>More Resources</a>: Code, theme, and add-on resources for Bootstrap
\ No newline at end of file
+<a href='http://blog.getbootstrap.com/' target='_blank' rel='nofollow'>Bootstrap Blog</a>: Bootstrap's offical blog<br/>
```

Comment:
Hi @JinHash5 

Please change the formatting of the Resource section to use Markdown links instead of HTML Anchor tags.

You shouldn't be adding `html` such as `<br />` to create linebreaks. Rather use `*` to denote a new item in a line.